examples/CMakeLists.txt

```diff
@@ -32,28 +32,8 @@ set (examples
     h5_elink_unix2win
     h5_shared_mesg
     h5_debug_trace
-    h5_vds
-    h5_vds-exc
-    h5_vds-exclim
-    h5_vds-eiger
-    h5_vds-simpleIO
-    h5_vds-percival
-    h5_vds-percival-unlim
-    h5_vds-percival-unlim-maxmin
 )
 
-if (H5_HAVE_PARALLEL)
-  set (parallel_examples
-    ph5example
-    ph5_filtered_writes
-    ph5_filtered_writes_no_sel
-  )
-
-  if (HDF5_ENABLE_SUBFILING_VFD)
```

Comment:
Looks like this particular piece is missing from the CMakeLists.txt file in the HDF5Examples directory

Reply:
I thought it was already in the PAR directory.

Reply:
Oh I see it's in C_sourcefiles.cmake instead